rules/integrations/google_workspace/persistence_application_added_to_google_workspace_domain.toml

```diff
@@ -25,7 +25,51 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Application Added to Google Workspace Domain"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Application Added to Google Workspace Domain
+
+Google Workspace Marketplace is an online store for free and paid web applications that work with Google Workspace services and with third party software. Listed applications are based on Google APIs or on Google Apps Script and created by both Google and third party developers.
+
+Marketplace applications require access to specific Google Workspace resources. Applications can be installed by individual users, if permissions grant availability or can be installed for an entire Google Workspace domain by administrators. Consent screens typically display what permissions and privileges the application requires during install. As a result, malicious Marketplace applications may require more permissions than necessary or have malicious intent.
+
+Google clearly states that they are not responsible for any product on the Marketplace that originates from a source other than Google.
+
+This rule identifies an application being manually added from the Marketplace by a Google Workspace account.
+
+#### Possible investigation steps
+
+- Identify user account(s) associated by reviewing `user.name` or `user.email` in the alert
+- This rule relies on data from `google_workspace.admin` thus indicating the associated user has administrative privileges to the Marketplace
+- With access to Google Workspace admin console, visit Security > Investigation tool with filters for the user email and event is `Assign Role` or `Update Role` to determine if new cloud roles were recently updated
+- With the user account, review other potentially related events within the last 48 hours
+- Re-assess the permissions and reviews of the Marketplace applications to determine if they violate organizational policies or introduce unexpected risks
+- With access to Google Workspace admin console, determine if the application was installed domain-wide or individually by visiting Apps > Google Workspace Marketplace Apps
```

Comment:
```suggestion
- Identify the user account(s) associated by reviewing `user.name` or `user.email` in the alert.
- This rule relies on data from `google_workspace.admin`, thus indicating the associated user has administrative privileges to the Marketplace.
- With access to the Google Workspace admin console, visit the `Security > Investigation tool` with filters for the user email and event is `Assign Role` or `Update Role` to determine if new cloud roles were recently updated.
- With the user account, review other potentially related events within the last 48 hours.
- Re-assess the permissions and reviews of the Marketplace applications to determine if they violate organizational policies or introduce unexpected risks.
- With access to the Google Workspace admin console, determine if the application was installed domain-wide or individually by visiting `Apps > Google Workspace Marketplace Apps`.
```

Reply:
```suggestion
- Identify the associated user accounts by reviewing `user.name` or `user.email` fields in the alert.
- This rule relies on data from `google_workspace.admin`, thus indicating the associated user has administrative privileges to the Marketplace.
- With access to the Google Workspace admin console, visit the `Security > Investigation tool` with filters for the user email and event is `Assign Role` or `Update Role` to determine if new cloud roles were recently updated.
- With the user account, review other potentially related events within the last 48 hours.
- Re-assess the permissions and reviews of the Marketplace applications to determine if they violate organizational policies or introduce unexpected risks.
- With access to the Google Workspace admin console, determine if the application was installed domain-wide or individually by visiting `Apps > Google Workspace Marketplace Apps`.
```

Reply:
A few notes/questions about this section:
- Lines 44 and 47: Are we telling users that they need to gain access to the Google Workspace admin console?
- Line 45: Are we telling users to review potentially related alerts that are from the same user account or are we suggesting something different?
- Line 46: Are we recommending that users re-asses the reviews of Marketplace applications? Or are we suggesting that they review Marketplace applications for policy violations and potential risks?